src/api/browser.ts

```diff
@@ -2,51 +2,43 @@ import EventEmitter from 'events'
 import { TSend } from '../protocol'
 import createPage from './page'
 
-const createBrowser = (send: TSend) => {
-  const eventEmitter = new EventEmitter()
+class Browser extends EventEmitter {
+  _send: TSend
```

Comment:
you don't need this, all of the ctor arguments are accessible from the class

Reply:
whaaat

Reply:
at least my TypeScript is complaining and I see why.

Reply:
I still don't get how args automagically appears in instance...

Reply:
take a look at this - https://www.stevefenton.co.uk/2013/04/stop-manually-assigning-typescript-constructor-parameters/